runtime/compiler/env/JITServerPersistentCHTable.hpp

```diff
@@ -63,6 +63,7 @@ class JITServerPersistentCHTable : public TR_PersistentCHTable
 
    virtual TR_PersistentClassInfo * findClassInfo(TR_OpaqueClassBlock * classId) override;
    virtual TR_PersistentClassInfo * findClassInfoAfterLocking(TR_OpaqueClassBlock * classId, TR::Compilation *, bool returnClassInfoForAOT = false) override;
+   virtual TR_PersistentClassInfo * findClassInfoAfterLocking(TR_OpaqueClassBlock * classId, TR_FrontEnd *, bool returnClassInfoForAOT = false) override;
```

Comment:
Since this new method is only used in `JITServerPersistentCHTable` maybe it should not be virtual, but marked private.

Reply:
It extends `TR_PersistentClassInfo * TR_PersistentCHTable::findClassInfoAfterLocking(TR_OpaqueClassBlock *classId, TR_FrontEnd *fe, bool returnClassInfoForAOT)` and used outside `JITServerPersistentCHTable ` by `TR_J9ServerVM::checkCHTableIfClassInfoExistsAndHasBeenExtended`.